styleguide/derek/pattern-components/resourceCenter-resource/refresh/_resourceCenter-resource.scss

```diff
@@ -0,0 +1,141 @@
+// Resource Icon Color
+@mixin resource-refresh($class, $color) {
+  #{$class} {
+    @include make-md-column(4);
+    @include make-sm-column(6);
+    background: $white;
+    box-shadow: 0 1px 1px 1px rgba(0, 0, 0, .06);
+    padding: 0;
+    transition: .3s ease-in-out;
+    width: 100%;
+
+    .resource-top {
+      background: $color;
+    }
+
+    a {
+      @include flex-box;
+      flex-flow: column nowrap;
+      height: 100%;
+      text-decoration: none;
+    }
+
+    &:hover {
+      box-shadow: 0 15px 25px -9px rgba(0, 0, 0, .2);
```

Comment:
Per your question in jira, all colors (even colors with various opacities) need to be in the swatches file.  That's the purpose of the swatches file, to keep us from creating too many color variations throughout the site.